### DISCUSSION TITLE
梅林 38X 固件 2.0 版本支持

### DISCUSSION THREAD

**Discussion by messense** [messense/aliyundrive-webdav#778] [Polls]
梅林固件路由器付费支持调研，通过付费加入[aliyundrive-webdav 知识星球](https://t.zsxq.com/0c9sq6Ca8)来支持梅林固件版本的开发和维护，并可以获得更好的问题咨询和技术支持服务。

目标：50个用户加入，达成目标后梅林包文件可对所有人开放，否则只对加入知识星球的会员开放。
BTW，知识星球上已提供了 2.1.0 版本的梅林 arm384+ aarch64 & armv7 测试版本下载，加入即可获取。

**2023/04/23 更新：感谢星球付费会员的支持，2.1.0 版本现已对所有人开放下载：https://github.com/messense/aliyundrive-webdav/releases/tag/v2.1.0 ，后续会优先给星球会员更新新版本。**

<img width="500" alt="image" src="https://user-images.githubusercontent.com/1556054/226156365-99a26dfd-b0c4-477c-b61c-fce2e700c474.png">

<img width="500" alt="qrcode" src="https://user-images.githubusercontent.com/1556054/226176140-d1714916-664c-41fc-abf0-74c149f8053d.png">

**Comment by wutao135** [comment]
100还是有点贵？为什么要对梅林固件单独收费呢

**Reply by messense** [reply]
> 为什么要对梅林固件单独收费呢

因为我不用梅林，没有义务免费开发、维护和答疑。

> 100还是有点贵？

可以选择不用。或者换 OpenWrt 也可以不用在这里付费。

**Reply by yxd987654321** [reply]
您好，openwrt路由器可以推荐个强点的吗？谢谢

**Comment by messense** [comment]
各位支持付费的小伙伴们：[知识星球](https://t.zsxq.com/0c9sq6Ca8)上目前已提供了 2.1.0 版本的梅林 arm384+ aarch64 & armv7 测试版本下载，加入即可获取。

**Reply by yxd987654321** [reply]
如果支持380就好了

**Reply by mnm158** [reply]
50个用户达成了吗 可以免费开放一下吗

**Comment by yang3737** [comment]
支持付费，但是100有点贵了，建议便宜点估计付费的人多一些

**Reply by sidfan32** [reply]
+1

**Reply by messense** [reply]
可以选择不用，或者把钱花在换 OpenWrt 路由器上也可以不用在这里付费。自由选择。

**Comment by williamyh313** [comment]
你好，请问知识星球里的，有对梅林380固件支持的阿里云插件吗

**Reply by messense** [reply]
没有，380 太老了不想支持了。

**Reply by yxd987654321** [reply]
> 没有，380太老了不想支持了。

您好，如果支持380点话，我愿意支持。

**Reply by aguero119** [reply]
386现在不支持了?

**Comment by xieyixi** [comment]
之前版本梅林也不能用了吗？

**Comment by michaeltsuining** [comment]
V8的merlin Clash插件也没有吗

**Reply by michaeltsuining** [reply]
给钱也可以呀

**Reply by messense** [reply]
不懂你什么在这里问 merlin Clash。

**Comment by EurekaMa** [comment]
现在梅林已经完全不能用了是不是

**Comment by yuanyuan428** [comment]
好像用不了了

**Comment by 421429567** [comment]
用不了啊，插件提示状态正常，但无法连接。

**Comment by DemonPresident** [comment]
请问是否有arm388的支持

**Comment by OROMISES** [comment]
安装的时候提示 缺少install.sh

**Comment by doubleyoung** [comment]
备份盘和资源盘会有新版本支持吗？
